DependencyInjection/Configuration.php

```diff
@@ -76,13 +89,11 @@ public function getConfigTreeBuilder()
                     ->addDefaultsIfNotSet()
                     ->children()
                         ->integerNode('token_ttl')
-                            ->cannotBeEmpty()
```

Comment:
This method cannot be used on IntegerNode, because it will throw an exception. I assume setting 0 as minimum value should do the trick.